doc-content/enterprise-only/fuse/ba-engines-karaf-con.adoc

```diff
@@ -0,0 +1,16 @@
+[id='ba-engines-karaf-con']
+= {PRODUCT} decision and process engines with {FUSE} on Apache Karaf
+
+Apache Karaf is a polymorphic standalone open-source container. It is based on the OSGi standard from the OSGi Alliance. Karaf provides support for modularisation through OSGi bundles with sophisticated class-loading support. You can deploy multiple versions of a dependency side by side in a Karaf container. You can use hot code swapping to upgrade or replace a module without shutting down the container.
+
+{PRODUCT} integration with {FUSE} on Karaf is provided through Karaf features. You can install individual components of {PRODUCT} for {FUSE} on Karaf using these features.
+
+The following features XML files facilitate {PRODUCT} and {FUSE} on Karaf integration:
+
+* `rhba-features-<FUSE-VERSION>-features.xml`
++
+This file is part of {FUSE} installed in Karaf where `<FUSE-VERSION>` is the version of Fuse. It is stored in the Karaf system repository, in the `system/org/jboss/fuse/features/rhba-features` directory. This file contains prerequisites for installing {PRODUCT} features.
+
+* `kie-karaf-features-{MAVEN_ARTIFACT_VERSION}-features-fuse.xml`
++
+This file is part of {PRODUCT} and provides {PRODUCT} features, which define the OSGi features that can be deployed into {FUSE_LONG}. OSGi users can install features from this file to install {PRODUCT} into {FUSE} and use it in their applications. You can find this features file in the online and offline Maven repository that is distributed with {PRODUCT}. The group ID, artifact, and version (GAV) of this file is `org.kie:kie-karaf-features:{MAVEN_ARTIFACT_VERSION}`.
```

Comment:
"artifact ID, ..."

And probably best to say "... (GAV) identifier of this ..."  I usually say either "identifier" if a user is looking for it, or "setting" if a user is listing it in a file. Reason is that the entire identifer/setting is a unit with specific syntax (colons, etc.) that matters, not just the individual GAV pieces.